lib/ykpiv.c

```diff
@@ -154,8 +156,7 @@ ykpiv_rc ykpiv_connect(ykpiv_state *state, const char *wanted) {
     }
     rc = SCardConnect(state->context, reader_ptr, SCARD_SHARE_SHARED,
         SCARD_PROTOCOL_T1, &state->card, &active_protocol);
-    if(rc != SCARD_S_SUCCESS)
-    {
+    if(rc != SCARD_S_SUCCESS) {
```

Comment:
Hi klail, thanks for the quick reply. For this "if" line, if you search for all 23 "if"s in the whole file, you can find that there are only 2 "if"s written in two lines, the rest 21 "if"s are all written as "if (...) {". So I personally suggest that we should keep a consistent style, but if you still wish me to drop this change, feel free to tell me.